source/standalone/workflows/ray/hyperparameter_tuning/vision_cartpole_cfg.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+import pathlib
+import sys
+
+# Allow for import of items from the ray workflow.
+CUR_DIR = pathlib.Path(__file__).parent
+UTIL_DIR = CUR_DIR.parent
+sys.path.extend([str(UTIL_DIR), str(CUR_DIR)])
```

Comment:
Similarly to above, this seems a bit hacky - is it necessary to modify the system path?

Reply:
again because standalone is not a python package? If there was a way to import this wouldn't need it